mint/git-clone/mint-leaf.yml

```diff
@@ -228,3 +234,53 @@ tasks:
       CHECKOUT_PATH: ${{ params.path }}
       GITHUB_TOKEN: ${{ params.github-access-token }}
       PRESERVE_GIT_DIR: ${{ params.preserve-git-dir }}
+
+  - key: generate-lfs-task
```

Comment:
Not a big deal, but as a small performance optimization we could pull this into the `git-clone` task to reduce the number of tasks and layers by 1.

Reply:
I think we actually don't even need this after the dynamic filter changes I made yesterday